definitions/control-v1.yaml

```diff
@@ -4538,6 +4553,20 @@ TOfReTlUQzgpXRW5h3n2LVXbXQhPGcVitb88Cm2R8cxQwgB1VncM8yvmKhREo2tz
           default: false
           description: If `true`, publishing messages with a push payload in the extras field is permitted and can trigger the delivery of a native push notification to registered devices for the channel.
           example: false
+        batchingEnabled:
+          type: boolean
+          default: false
+          description: If `true`, channels within this namespace will start batching inbound messages instead of sending them out immediately to subscribers.
+        batchingPolicy:
+          type: string
+          default: simple
+          description: If `true`, groups multiple incoming messages into a single batch.
+          example: simple
+        batchingInterval:
+          type: integer
+          default: simple
```

Comment:
Same here - should be 20 as its an integer. Same for its example (though the example doesn't have to be 20).